### DISCUSSION TITLE
https://olivierbinette.github.io/posts/2022-01-01-record-linkage-at-the-gpsg-community-pantry/

### DISCUSSION THREAD

**Discussion by giscus** [OlivierBinette/olivierbinette.github.io#1] [Announcements]
# https://olivierbinette.github.io/posts/2022-01-01-record-linkage-at-the-gpsg-community-pantry/

The Duke Graduate and Professional Student Government (GPSG) Community Pantry is a student-operated food pantry serving the student community at Duke University. In this post, I describe the record linkage system used at the Pantry to identify individual customers and obtain their order history. This is done using a Python module for deterministic record linkage and model evaluation techniques which I describe in detail.

https://olivierbinette.github.io/posts/2022-01-01-record-linkage-at-the-gpsg-community-pantry/

**Comment by JeffJonas** [comment]
Hey that's pretty slick. BTW: If you want to run the same data (locally on your computer) through the Senzing algorithm -- just to see the differences -- you can download the Senzing desktop app here: https://senzing.com/desktop/

Would love to hear any feedback, either way! jeff@senzing.com THX

**Reply by OlivierBinette** [reply]
Thanks for the suggestion! I only have a Linux laptop though :/ I'll try to borrow a friend's computer to run this.

**Comment by JeffJonas** [comment]
Coolio!

Drop a note to ***@***.******@***.***> if you run into any trouble, have questions, etc.


Jeff Jonas
Senzing, Founder and CEO
